app/controllers/product_controller.rb

```diff
@@ -0,0 +1,24 @@
+class ProductController < ApplicationController
+  def new
+    @product = Product.new
+  end
+
+  def create
+    product = Product.create(product_params)
+    redirect_to products_path
```

Comment:
It seems that you don't use the `product` variable here so maybe you can just have `Product.create(product_params)` ?